streams/src/test/java/org/apache/kafka/streams/state/internals/AbstractKeyValueStoreTest.java

```diff
@@ -81,6 +82,16 @@ public void after() {
         return result;
     }
 
+    private static List<KeyValue<Integer, String>> getOrderedContents(final KeyValueIterator<Integer, String> iter) {
+        final LinkedList<KeyValue<Integer, String>> result = new LinkedList<>();
+        while (iter.hasNext()) {
+            final KeyValue<Integer, String> entry = iter.next();
+            result.add(entry);
+        }
+
+        return result;
+    }
```

Comment:
This idiom is used across a bunch of the tests, which is fine, but it's also a pretty generic bit of logic. I wonder if it would be better to move it to TestUtils. In fact, there are already a couple of utilities there for comparing iterators, and I bet there are also Hamcrest matchers that compare iterators.

I'm fine with this as-is, if you prefer it; I just wondered if you knew about those other options.

Reply:
Makes sense. I changed to the code to use existing util functions whenever possible. If you think the changes overshoot let me know, I'm also not entirely sure.